src/components/PostComment.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext, useState } from "react";
+import { useParams } from "react-router-dom/dist";
+import { postComment } from "../utils/api";
+import { UserContext } from "../contexts/users";
+
+export default function PostComment({ setNewComment }) {
+  const [newBody, setNewBody] = useState("");
+  const { user } = useContext(UserContext);
+  const { review_id } = useParams();
+  const [error, setError] = useState("");
+  const [err, setErr] = useState(null);
+

```

Comment:
I enjoy how you've handled figuring out how to fire of the useEffect to make a fresh request for comments based on setNewComment. However, I would rename the values of newComment and setNewComment. You could call then something a little verbose like newCommentCount to describe what you are doing or you could even consider a holding a boolean value in state to do similar logic and call it addNewComment or something. This would just make it even more readable!